src/main/java/mods/natura/items/PlantItem.java

```diff
@@ -65,15 +65,12 @@ public void addInformation(ItemStack stack, EntityPlayer player, List list, bool
                 list.add(StatCollector.translateToLocal("tooltip.cotton"));
                 break;
             case 4:
-                list.add(StatCollector.translateToLocal("tooltip.sulfur"));
-                break;
-            case 5:
                 list.add(StatCollector.translateToLocal("tooltip.fletching"));
                 break;
-            case 6:
+            case 5:
```

Comment:
Just because you removed a case doesn't mean these need to be changed, this will cause wrong tooltips on the other items